blog/_posts/btcpay-server-1-5-0.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Introducing BTCPay Server 1.0.5 - Dashboard and payout processors"
+date: 2022-04-28
+author: pavlenex
+category:
+  - "releases"
+tags:
+  - "btcpay-server"
+  - "release"
+  - "payouts"
+  - "dashboard"
+  - "payout processors"
+  - "update"
+coverImage: "/images/BTCPay1-5-0-desk.png"
+---
+
+After a long wait, we're finally out of our coding cave, ready to introduce a new major BTCPay Server version - 1.5.0! 
+
+This release brings in a brand-new dashboard for easier analytics and better data visualization, payout processors and schedule for later for better payout management payments, batching your payments and refunds, several UX/UI improvements, greenfield API additions and a few non-critical bug fixes.
+
+## New dashboard 📊
+
+We know that having a quick glance at your store's performance is essential. Also we understand that as a user, you would like to have an easy way to access and manage recent invoices and refunds. That's why we're introducing a **dashboard**!
```

Comment:
```suggestion
We know that having a quick glance at your store's performance is essential. We also heard you about easy access and management of recent invoices and refunds. This is where we introduce you to your new dashboard!
```

Reply:
Would commit, but this doesn't make sense

> This is where we introduce you to your new dashboard!

Probably meant `why`

Reply:
```suggestion
We know that having a quick glance at your store's performance is essential. We also heard you about easy access and management of recent invoices and refunds. With that in mind, we'd like to introduce you to your store's new **dashboard**!
``` 
No didn't mean Why, it makes sense to me as we already had the the Dashboard, but now we introduce them to a new layout (functionality) of that. But maybe it makes more sense this way :)